UPGRADE-2.8.md

```diff
@@ -511,5 +511,5 @@ Yaml
 
  * Deprecated usage of a colon in an unquoted mapping value
  * Deprecated usage of `@`, `` ` ``, `|`, and `>` at the beginning of an unquoted string
- * Deprecated non-escaped \ in double-quoted strings when parsing Yaml
-   ("Foo\Var" is not valid whereas "Foo\\Var" is)
+ * Deprecated non-escaped `\` in double-quoted strings when parsing Yaml
+   (`Foo\Var` is not valid whereas `Foo\\Var` is)
```

Comment:
Exactly, escaping the backslash is only needed when using double quotes. [Escape sequences are only available in double-quoted strings](http://www.yaml.org/spec/1.2/spec.html#id2776092):

>  Note that escape sequences are only interpreted in double-quoted scalars. In all other scalar styles, the “\” character has no special meaning and non-printable characters are not available.